frontend/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.5.1",
+    "@types/jest": "24.0.19",
```

Comment:
Not too important, but make sure you keep devDependencies out of the depencies prop. It will bloat the `npm install --production` command. Generally, `@types` depemdencies are devDependencies.

Reply:
I agree, my bad. It was added to dependencies instead of devDependencies while creating the app using ```create-react-app```.